src/xenia/ui/gl/gl_immediate_drawer.cc

```diff
@@ -270,6 +270,16 @@ void GLImmediateDrawer::End() {
   }
 }
 
+void GLImmediateDrawer::EnableAlphaTest(bool enable) {
```

Comment:
side-channeling this is not good - make it part of the API if it's required. add `bool alpha_blend` to ImmediateDraw, have imgui_drawer populate it, and then have the gl drawer ::Draw switch its mode based on that (being smart about only switching if needed)